templates/download/server/choose.html

```diff
@@ -79,14 +116,30 @@ <h2>Alternative downloads</h2>
   </div>
   <div class="row u-equal-height">
     <div class="col-4 p-card">
-      <h3 class="p-card__title">Ubuntu Server for ARM</h3>
-      <p>The next version of Ubuntu Server, to be officially released in October 2020.</p>
-      <p><a href="http://cdimage.ubuntu.com/ubuntu-server/daily-live/current/" class="p-link--external">Get Ubuntu Server 20.10</a></p>
+      <h3 class="p-card__title">BitTorrents</h3>
+      <p>BitTorrent sometimes enables higher download speeds and more reliable downloads of large files.</p>
+      <ul class="p-list">
+        <li class="p-list__item">
+          <a class="download-torrent" href="https://releases.ubuntu.com/{{ releases.lts.short_version }}/ubuntu-{{ releases.lts.full_version }}-live-server-amd64.iso.torrent">
+          Ubuntu Server {{ releases.lts.full_version }} LTS&nbsp;&rsaquo;
+        </a>
+        </li>
+        <li class="p-list__item">
+          <a class="download-torrent" href="https://releases.ubuntu.com/{{ releases.latest.short_version }}/ubuntu-{{ releases.latest.full_version }}-live-server-amd64.iso.torrent">
+          Ubuntu Server {{ releases.latest.full_version }}&nbsp;&rsaquo;
+        </a>
+        </li>
+    </ul>
     </div>
     <div class="col-4 p-card">
-      <h3 class="p-card__title">Ubuntu Server 18.04 LTS</h3>
-      <p>Ubuntu is available on the IBM POWER platform, bringing the entire Ubuntu and OpenStack ecosystem to IBM POWER.</p>
-      <p><a href="http://cdimage.ubuntu.com/ubuntu/releases/18.04/release/" class="p-link--external">Get Ubuntu Server 18.04 LTS</a></p>
+      <h3 class="p-card__title">Ubuntu&nbsp;Server&nbsp;{{ releases.previous_lts.short_version }}&nbsp;LTS</h3>
```

Comment:
Using `&nbsp;` causes small screen to not wrap:
![image](https://user-images.githubusercontent.com/1413534/96655409-4bbffd80-1335-11eb-8726-cd6626bc1c19.png)